plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestTrinoRestCatalogConnectorTest.java

```diff
@@ -176,4 +179,57 @@ public void testMaterializedViewSnapshotSummariesHaveTrinoQueryId()
         assertThatThrownBy(super::testMaterializedViewSnapshotSummariesHaveTrinoQueryId)
                 .hasMessageContaining("createMaterializedView is not supported for Iceberg REST catalog");
     }
+
+    @Override
```

Comment:
Whevenever overriding a test, document  why you're doing that.
Reviewers and future code readers will want to remove the (unnecessary) override, so it's nice to let know what was the point.

Also,.... we should try avoid the override.

Reply:
I've tried to mention it in L219, but that's probably too difficult to find when skimming the code. I'll make sure that it's more obvious why we're overriding this (which is due to https://github.com/apache/iceberg/pull/6511)

Reply:
Why not call `purgeTable` method likes #15855? The new `dropTable` behavior is different from the expected behavior as Trino.